e2e/tests/update_params_test.go

```diff
@@ -109,112 +106,55 @@ func (s *suite) AnAuthzGrantFromTheAccountToTheAccountForTheMessage(granterName,
 		} `json:"grants"`
 	}
 
-	err = json.Unmarshal([]byte(res.Stdout), &grantsRes)
+	// Unmarshal the JSON response into the grantsRes struct.
+	responseBz := []byte(strings.TrimSpace(res.Stdout))
+	err = json.Unmarshal(responseBz, &grantsRes)
 	require.NoError(s, err)
 
+	// Check if the expected grant is found in the response.
 	grantFound := false
 	for _, grant := range grantsRes.Grants {
+		s.Log(grant)
```

Comment:
```suggestion
	s.Log("Authz grants:")
	grantFound := false
	for _, grant := range grantsRes.Grants {
		s.Log(grant)
```

OR

```suggestion
	s.Logf("Authz grants: %s", res.Stdout)
	grantFound := false
	for _, grant := range grantsRes.Grants {
```

OR

```suggestion
	grantFound := false
	for _, grant := range grantsRes.Grants {
```

Reply:
@Olshansk WDYT here? Did you intentionally leave the log line in or can we remove it?

Reply:
Let's remove the logline.